### PR TITLE
Gf 393 any

### PR DESCRIPTION
**Pull request by jhphan**
This merge includes the following fixes: the GF-393 bug fix for the Any datatype with associated documentation updates, a quoting fix in the wrapper script that was missed from a previous bug. It also includes an update to the Docker build scripts, and the tutorial outlines.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Build Docker Image
+
+Use the folloiwng steps to run build a GeneFlow docker image:
```
Comment (pierreeraymond) [inline docker/README.md]: Incorrect spelling of "following".  Should "to run build" fragment actually be "to run and build", or "to run a build"?

File: docs/chapters/apps.rst [hunk 2]
```diff
@@ -111,7 +111,7 @@ test_value:
   (Optional) If specified, used as the value for this input/parameter when generating the test script.
```
Comment (pierreeraymond) [inline docs/chapters/apps.rst]: Should the word "used" be "use" instead?

File: docker/README.md (hunk 1 above)
Comment (jhphan) [reply docker/README.md]: Fixed spelling.

File: docs/chapters/apps.rst (hunk 2 above)
Comment (jhphan) [reply docs/chapters/apps.rst]: Re-worded that sentence.

Comment (pierreeraymond) [reply docs/chapters/apps.rst]: Resolving

**Pull request merged**